### ISSUE TITLE
Question concerning loss function in the paper

### ISSUE THREAD

**Issue by xSeanliux** [auspicious3000/autovc#35]
Hello, 
I'd like to enquire on the loss function mentioned in section 3.2, under the section "Training", Equation (6), where it mentions the definition of the loss function. I'd like to ask two questions concerning the two equations: 

1. Do the \mathbb{E} brackets denote expected values? If so, how is it calculated? (There is only one Xhat1  and X1 per iteration)
2. What do the indices outside of the absolute values mean? I take the absolute values to mean the L2 norm of the pointwise differences of the tensors, and the 2 on top of the definition of L_recon to mean power (that is, norm squared). 

Thank you for your time!
